kubejs/server_scripts/base/unification/unify_materials.js

```diff
@@ -7,7 +7,7 @@ onEvent('recipes', (event) => {
         let nugget = getPreferredItemInTag(Ingredient.of(`#forge:nuggets/${material}`)).id;
 
         let gem = getPreferredItemInTag(Ingredient.of(`#forge:gems/${material}`)).id;
-        let chunk = getPreferredItemInTag(Ingredient.of(`#forge:chunks/${material}`)).id;
+        let chunk = getPreferredItemInTag(Ingredient.of(`#forge:raw_materials/${material}`)).id;
```

Comment:
What is your intention with this change? :)

Reply:
The new tag is raw_materials not chunk. I just feel like I need to change it so that maybe in the future it's convenient to work with. Besides, just out of my curiosity, why is it there in the first place :D?

Reply:
Because our oregen mod in 1.16.5 introduced chunks as output from ores, sadly it's not on 1.18.2 so it's pointless to have in the scripts 😅

Reply:
I'd prefer you reverted the change here, to make sure it doesn't start producing recipes :P